packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -47,6 +47,17 @@ enum _RefreshIndicatorMode {
   canceled, // Animating the indicator's fade-out after not arming.
 }
 
+/// Used to configure the [RefreshIndicator] trigger mode.
+enum RefreshIndicatorTriggerMode {
+  /// The indicator can trigger when the child's [Scrollable] descendant overscrolls
+  /// by drag(not inertia), regardless of whether its initial scroll position is on the edge or not.
+  loose,
+
+  /// The indicator can only trigger when the child's [Scrollable] descendant overscrolls
```

Comment:
suggestion: The indicator can only be triggered if the [Scrollable] is at the edge when the drag starts.